gnosis/conditional-markets-interface
# Feature/#89upgrade from tier is pending

# Description
This PR adds feedback for an user upgrading to tier 2 in markets view. It includes some messages in trading limit indicator and also changes to add messages in "TradeOverLimit" modal and a new modal "MyAccount" that can be open clicking over Tier in limit indicator (maybe change of click spot when finally decided)

# Code
This PR will:

- Create a new "MyAccount" modal to show details for an account and also Tier upgrade feedback.
- Modify "TradeOverLimit" modal to show Tier upgrade feedback
- Modify "TierLimitIndicator" to show user feedback when upgrading
- Add a new step in Tier 2 KYC flow to help an user when SumSub token expired

# To Test
- Go to any Sight market with an unregistered Ethereum account and create a new account.
  - Make sure to use a legit email, as this process will sent unique codes there.
  - This can be tested from Tier 0 to Tier 2 or Tier 1 to Tier 2
- Email link will use "development" environment, take this into consideration.
- Check any market to see that "TierLimitIndicator" should show a message to encourage to finish the process
  - If more details are required "MyAccount" modal will pop-up.
  - We can try to ask again for the email and see what happen
- We can test to do a really big trade and "TradeOverLimit" modal should pop-up
  - We can also execute the flow to request the email again and see what happen
- We should complete KYC and check once the process is finished use still gets feedback in market view.
- Final verification must be done manually in SumSub test aplication.
